packages/avatars/src/styled/StyledStandaloneStatusIndicator.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+import { IStyledStatusIndicatorProps } from './utility';
+import { StyledStatusIndicatorBase } from './StyledStatusIndicatorBase';
+
+const COMPONENT_ID = 'avatars.status-indicator.indicator';
+
+export const StyledStandaloneStatusIndicator = styled(StyledStatusIndicatorBase).attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})<IStyledStatusIndicatorProps>`
+  position: relative;
+  margin-top: ${props => `${props.theme.space.base / (props.size === 'small' ? 1 : 2)}px`};
```

Comment:
```suggestion
  margin-top: ${props => `calc((${props.theme.lineHeights.md} - ${props.theme.fontSizes.md}) / 2)`};
```

as this would respond more accurately to theming modifications. Another option would be to export `lineHeight` and `fontSize` values from `StyledStandaloneStatusCaption` and use them here. Perhaps another would be to move the return values out to a local `utility` function and use them in both places. 🤷

Reply:
Having a shared utility here does make more sense, thanks for spotting this. I will update this shortly.

Reply:
After a bit of trial, I found the unitless return value of `getLineHeight` didn't work well in translating over to pixels for `margin-top` calculations - . As for exporting `lineHeight` and `fontSize` values, those are based on the theme and are calculated at runtime (as opposed to static values), unless we use `props.theme.lineHeights.md` directly.

What I found did work was `calc((${props.theme.lineHeights.md} - ${getStatusSize(props, '0')}) / 2)`.